src/postman/__tests__/operation.test.ts

```diff
@@ -0,0 +1,89 @@
+import { HeaderDefinition, RequestBody } from 'postman-collection';
+import { transformPostmanCollectionOperation } from '../operation';
+
+describe('transformPostmanCollectionOperation()', () => {
+  describe('operation can be found', () => {
+    describe('description is set', () => {
+      it('returns operation with description', () => {
+        expect(
+          transformPostmanCollectionOperation({
+            document: {
+              item: [
+                {
+                  request: {
+                    method: 'get',
+                    url: '/path/:param?a=b',
+                    body: { mode: 'raw', raw: 'test' } as RequestBody,
+                    header: [{ key: 'header', value: 'a header' }] as HeaderDefinition,
+                  },
+                  description: 'desc',
+                },
+              ],
+            },
+            method: 'get',
+            path: '/path/{param}',
+          }),
+        ).toEqual({
+          description: 'desc',
+          id: '?http-operation-id?',
+          iid: expect.stringMatching(/^[a-z0-9]{8}-([a-z0-9]{4}-){3}[a-z0-9]{12}$/),
+          method: 'get',
+          path: '/path/{param}',
+          request: expect.any(Object),
```

Comment:
Hmm, can't we be a little bit more precise about this assertion?

Reply:
Well, I decided not to verify the whole request object because it is huge and it is already verified precisely together with `transformRequest` test. However, I will not defend that. WDYT? Test it all?

Reply:
Hmm, hmm…I understand the reasoning, I guess we can get away with it.